### DISCUSSION TITLE
Delay in updating line/column counters

### DISCUSSION THREAD

**Discussion by zunino** [nvim-lualine/lualine.nvim#801] [General]
Recently, I have realized there's a noticeable delay in updating the line and column counters in response to moving the cursor around. As a matter of fact, the same thing also happens to the other field that indicates the cursor's position relative to the buffer, in percentage. Does anybody know why this is the case? Is this some kind of event debouncing that's taking place? Or is it something else?

Thank you.

**Comment by fitrh** [comment]
It is from recent update, you can configure it using `options.refresh.statusline`  configuration, the default is 1000ms

**Reply by zunino** [reply]
Bullseye! I set it to 100ms and my editor seems "alive" again. Thank you!

**Comment by shadmansaleh** [comment]
Are you on latest commit ? 

With lualline updating on CursorMoved this should be a non issue https://github.com/nvim-lualine/lualine.nvim/commit/664c688974a7f183f6d6edae70c6aa117dcbd009

**Reply by zunino** [reply]
I checked and the commit is indeed in my local repository. I then removed the explicit `refresh` option I had added to the lualine config and the counters were still updating instantly as you suggested. I suppose I hadn't `Packer Update`d the first time around. Thank you.
